javascript/async-apis/APIs.md

```diff
@@ -78,10 +78,12 @@ More recently, however, web browsers have begun to implement a new native functi
 ~~~javascript
 // URL (required), options (optional)
 fetch('https://url.com/some/url')
-  .then(function(response) {
-}).catch(function(err) {
-  // Error :(
-});
+  .then(function(response) { 
+  
```

Comment:
For line 82, can you add a comment block about, maybe something like `// Successful response :)`. Something similar to line 85.